tasks/cachebust.js

```diff
@@ -80,10 +83,14 @@ module.exports = function() {
         }
 
         function addFileHash(str, hash, separator) {
-            var parsed = url.parse(str);
-            var ext = path.extname(parsed.pathname);
+            if (opts.useUrlParam) {
+                return str + '?cache-bust=' + hash;
```

Comment:
Don't believe the `cache-bust` section is needed, so can we remove that and just have the hash